hypha/apply/users/forms.py

```diff
@@ -164,7 +164,7 @@ class Meta:
     def __init__(self, *args, **kwargs):
         self.request = kwargs.pop("request", None)
         super().__init__(*args, **kwargs)
-        if not self.instance.is_apply_staff_or_finance:
+        if not (self.instance.is_apply_staff or self.instance.is_finance):
```

Comment:
Can we not use the new `is_org_faculty` here?

Reply:
Yes! this makes more sense too as contracting will likely be using slack. I just did a blind substitution after removing `is_apply_staff_or_finance`